src/sidebar/host-config.js

```diff
@@ -50,13 +50,45 @@ export default function hostPageConfig(window) {
     'usernameUrl',
   ];
 
+  // Coerces specific config values that we expect to be of specific types
+  // but could be provided in another.
+  const translations = {
```

Comment:
I think "coercions" or "casts" would be a preferable term here as it is more specific than "translation".